bin/getCryptoData.js

```diff
@@ -0,0 +1,162 @@
+import path from 'path';
+import fetch from 'node-fetch';
+import Promise from 'bluebird';
+import fs from 'fs';
+import jsonformat from 'json-format';
+import draftLog from 'draftlog';
+
+const iconOutputPath = `${__dirname}${path.sep}..${path.sep}imgs${path.sep}cryptoIcons${path.sep}/`;
+const translationFile = `${__dirname}${path.sep}..${path.sep}js${path.sep}` +
+  `languages${path.sep}/en_US.json`;
+
```

Comment:
Below is just an initial whitelist. Once Tyler implements his, I'll beef this one up. For now we could manually manage it, but I suppose if Tyler exposes an endpoint whereby we could obtain the whitelist, then I could work that process right into the script.

In any case, this script should probably be run at least once per release. If it finds any new/updated icons and/or coin names, the appropriate files will be updated and a PR could be made to merge them in.